src/snowcli/cli/snowpark/compute_pool/commands.py

```diff
@@ -25,7 +43,8 @@ def create(
         ...,
         "--family",
         "-f",
-        help="Name of the instance family. For more information about instance families, refer to the SQL CREATE COMPUTE POLL command.",
+        click_type=click.Choice(INSTANCE_FAMILY_OPTIONS),
```

Comment:
Instead of that you can do `instance_family: InstanceFamily`, typer will automatically make it a choice field

Reply:
oh, nice!